yarn.lock

```diff
@@ -5093,6 +5093,14 @@
   resolved "https://registry.yarnpkg.com/@segment/analytics-react-native/-/analytics-react-native-1.4.4.tgz#1c5e4f18ac46dd899947c758f104004fc7d5ffce"
   integrity sha512-zSgoGvw3VNqam0QoumOGiNr97ar3tutSwuMJPj2Da6mcJBVohMYHQcpShyufWfsiDeX8oMnu+UJx5eo+cxskrw==
 
+"@segment/loosely-validate-event@^2.0.0":
```

Comment:
Make sure that the `yarn.lock` for the specific package is updated as well or you may face deployment issues later. 

@jeanregisser explained to to me you can do that by:
1. Manually changing the workspaces in the root packages.json to `"packages": [ "packages/!(<YOUR_PACKAGE>)*"]`
2. Run yarn within your package directory
3. Change the workspaces back as it was